source/includes/gho/_api.md

```diff
@@ -0,0 +1,302 @@
+# Queries
+## employee \([Employee](#employee)\)
+An Onboarding employee record
+
+Argument | Type | Description
+--------- | ----------- | -----------
+id | [Int](#int) | 
+## employees \([EmployeeConnection](#employeeconnection)\)
+A collection of Onboarding employee records
+
+Argument | Type | Description
+--------- | ----------- | -----------
+first | [Int](#int) | Returns the first _n_ elements from the list.
+after | [String](#string) | Returns the elements in the list that come after the specified global ID.
+last | [Int](#int) | Returns the last _n_ elements from the list.
+before | [String](#string) | Returns the elements in the list that come before the specified global ID.
+start_date | [DateFilter](#datefilter) | 
+date_of_birth | [DateFilter](#datefilter) | 
+with_custom_field_value | [WithCustomFieldValue](#withcustomfieldvalue) | 
+# Mutations
+## updateEmployeeProfile \([Employee](#employee)\)
+Update an employee's profile
+
+Argument | Type | Description
+--------- | ----------- | -----------
+id | [ID](#id) | 
+employee_updates | [UpdateEmployee](#updateemployee) | 
+# Types
+## CustomField
+    Represents a single CustomField record for your company.  CustomFields can be stored and displayed in a variety of    ways.  The types are described via the [CustomFieldTypes](#customfieldtypes) enum.  
```

Comment:
If you [View](https://github.com/grnhse/greenhouse-api-docs/blob/8e4047f26bc0360dbb971b599ccee3a0a5c713aa/source/includes/gho/_api.md) this file, this line is coming up as a code block. I think you have to delete the four spaces at the front of this line for it to not do that. You probably should delete the four spaces between "of" and "ways" too.